design/sketch.webidl

```diff
@@ -368,13 +368,11 @@ interface WebGPUInputStepMode {
 
 dictionary WebGPUVertexAttributeDescriptor {
     u32 shaderLocation;
-    u32 inputSlot;
+    u32 vertexBufferIndex;
     u32 offset;
-    WebGPUVertexFormatEnum format;
```

Comment:
We need to keep the format because for example a shader taking vec2 as an input can have data stored at in many different ways:

 - As two 8/16/32 bit integers that are converted to a float with the same value.
 - As two 8/16/32 bit integers that are "normalized" to be turned in a float in the [0, 1] range.
 - As two float.